src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsSaveButtonV2.tsx

```diff
@@ -0,0 +1,94 @@
+import { ContextModule } from "@artsy/cohesion"
+import { ArtworkActionsWatchLotButtonFragmentContainer } from "Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActionsWatchLotButton"
+import { SaveUtilButton } from "Apps/Artwork/Components/ArtworkImageBrowser/SaveUtilButton"
+import { useArtworkLists } from "Components/Artwork/useArtworkLists"
+import { ProgressiveOnboardingSaveArtworkQueryRenderer } from "Components/ProgressiveOnboarding/ProgressiveOnboardingSaveArtwork"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import createLogger from "Utils/logger"
+import { ArtworkActionsSaveButtonV2_artwork$data } from "__generated__/ArtworkActionsSaveButtonV2_artwork.graphql"
+
+const logger = createLogger("ArtworkActionsSaveButtonV2")
+
+interface ArtworkActionsSaveButtonV2Props {
+  artwork: ArtworkActionsSaveButtonV2_artwork$data
+}
+
+export const ArtworkActionsSaveButtonV2: FC<ArtworkActionsSaveButtonV2Props> = ({
+  artwork,
+}) => {
+  const { isAuction, isClosed } = artwork.sale ?? {}
+  const isOpenSale = isAuction && !isClosed
+  const customListsCount = artwork.customCollections?.totalCount ?? 0
+  const isSavedToCustomLists = customListsCount > 0
+
+  const { isSaved, saveArtworkToLists } = useArtworkLists({
+    contextModule: ContextModule.artworkImage,
+    artwork: {
+      internalID: artwork.internalID,
+      id: artwork.id,
+      slug: artwork.slug,
+      title: `${artwork.title}, ${artwork.date}`,
+      imageURL: artwork.preview?.url ?? null,
+      isSavedToDefaultList: !!artwork.isSaved,
+      isSavedToCustomLists: isSavedToCustomLists,
+      isInAuction: !!isOpenSale,
+    },
+  })
+
+  const handleSave = async () => {
+    try {
+      await saveArtworkToLists()
+    } catch (error) {
+      logger.error(error)
+    }
+  }
+
+  if (isOpenSale) {
+    return (
+      <ArtworkActionsWatchLotButtonFragmentContainer
+        isSaved={isSaved}
+        artwork={artwork}
+        onClick={handleSave}
+        canShowRegistrationPopover={!isSavedToCustomLists}
+      />
+    )
+  }
+
+  return (
+    <ProgressiveOnboardingSaveArtworkQueryRenderer>
```

Comment:
@dzucconi is it okay that I placed `ProgressiveOnboardingSaveArtworkQueryRenderer` here?

Reply:
Yeah, this is fine. It was moved to avoid targeting auction artworks.